library/pod-security-policy/selinux/template.yaml

```diff
@@ -16,17 +16,19 @@ spec:
           properties:
             allowedSELinuxOptions:
               type: array
+              description: "An allow-list of seLinuxOptions configurations."
               items:
                 type: object
+                description: "A single, allowed configuration of seLinuxOptions for a pod container."
                 properties:
                   level:
-                      type: string
+                    type: string
```

Comment:
missing descriptions for these fields

Reply:
These are actually hard to find a succinct description of without saying something meaningless like "An SELinux level".  

Kubernetes has these same kind of meaningless descriptions in [their code](https://github.com/kubernetes/api/blob/87c4113e35a1b376e4434139109105243860ea52/core/v1/types.go#L6220). 

Is something like `An SELinux level.` ok for this use-case?  Not sure if we need to be responsible for teaching people how to do SE Linux, particularly given that I don't know anything about it 😂

Reply:
Yeah, a reference to some other (presumably authoritative) concept is sufficient. Teaching others about implementation details of systems we don't maintain is likely out-of-scope.

Reply:
Ok.  I've added some short descriptions like that to the 4 fields.